### ISSUE TITLE
Speaker notes timer stops working when opened twice

### ISSUE THREAD

**Issue by hakimel** [hakimel/reveal.js#2229]
To reproduce:
1. Press S to open the speaker view (timer works)
2. Press S to open the speaker view again while it's still open (timer stops)

Discovered in #1922

**Comment by oyron** [comment]
When pressing S the second time (Notes window already existing), the code in `plugin/notes/notes.js` reopens the existing notes window. The timer does not work because the pages renders immediately after `window.open()`, before the `Reveal` property is set. See the code below.

A better solution would be to give focus to the Notes window if it exists, rather than reloading it. This would also preserve the timer value. I have created a pull request for this.

```javascript
function openNotes( notesFilePath ) {

	if( !notesFilePath ) {
		var jsFileLocation = document.querySelector('script[src$="notes.js"]').src;  // this js file path
		jsFileLocation = jsFileLocation.replace(/notes\.js(\?.*)?$/, '');   // the js folder path
		notesFilePath = jsFileLocation + 'notes.html';
	}

	var notesPopup = window.open( notesFilePath, 'reveal.js - Notes', 'width=1100,height=700' );

	if( !notesPopup ) {
		alert( 'Speaker view popup failed to open. Please make sure popups are allowed and reopen the speaker view.' );
		return;
	}

	// Allow popup window access to Reveal API
	notesPopup.Reveal = window.Reveal;

	...
```

**Comment by hakimel** [comment]
I've merged in the fix in #2251 🙌
